app/components/EventItem/index.js

```diff
@@ -57,45 +59,72 @@ type EventItemProps = {
   field?: EventTimeType,
   showTags?: boolean,
   loggedIn: boolean,
+  eventStyle?: EventStyle,
 };
 
 const EventItem = ({
   event,
   field = EVENTFIELDS.start,
   showTags = true,
   loggedIn = false,
-}: EventItemProps): Node => (
-  <div
-    style={{ borderColor: colorForEvent(event.eventType) }}
-    className={styles.eventItem}
-  >
-    <div>
-      <Link to={`/events/${event.id}`}>
-        <h3 className={styles.eventItemTitle}>{event.title}</h3>
-        {event.totalCapacity > 0 && (
-          <Attendance
-            registrationCount={event.registrationCount}
-            totalCapacity={event.totalCapacity}
-            event={event}
-          />
-        )}
-      </Link>
-      <TimeStamp event={event} field={field} loggedIn={loggedIn} />
-      {showTags && (
-        <Flex wrap>
-          {event.tags.map((tag, index) => (
-            <Tag key={index} tag={tag} />
-          ))}
-        </Flex>
-      )}
-    </div>
+  eventStyle,
+}: EventItemProps): Node => {
+  switch (eventStyle) {
+    case 'extra-compact':
+      return (
+        <div
+          style={{ borderColor: colorForEvent(event.eventType) }}
+          className={styles.eventItem}
+        >
+          <div>
+            <Link to={`/events/${event.id}`}>
+              <h4 className={styles.eventItemTitle}>{event.title}</h4>
+            </Link>
+            <Time time={event.startTime} format="ll - HH:mm" />
+          </div>
+          <div className={styles.companyLogoExtraCompact}>
+            {event.cover && (
+              <Image src={event.cover} placeholder={event.coverPlaceholder} />
+            )}
+          </div>
+        </div>
+      );
 
-    <div className={styles.companyLogo}>
-      {event.cover && (
-        <Image src={event.cover} placeholder={event.coverPlaceholder} />
-      )}
-    </div>
-  </div>
-);
+    default:
+      return (
+        <div
+          style={{ borderColor: colorForEvent(event.eventType) }}
+          className={styles.eventItem}
+        >
+          <div>
+            <Link to={`/events/${event.id}`}>
+              <h3 className={styles.eventItemTitle}>{event.title}</h3>
+              {event.totalCapacity > 0 && (
+                <Attendance
+                  registrationCount={event.registrationCount}
+                  totalCapacity={event.totalCapacity}
+                  event={event}
+                />
+              )}
+            </Link>
+            <TimeStamp event={event} field={field} loggedIn={loggedIn} />
+            {showTags && (
+              <Flex wrap>
+                {event.tags.map((tag, index) => (
+                  <Tag key={index} tag={tag} />
+                ))}
+              </Flex>
+            )}
+          </div>
+
+          <div className={styles.companyLogo}>
```

Comment:
Turn this `div` into a `Flex`, and remove the unnecessary CSS. Same goes for the compact version above.